<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>481</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 136, 136);</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>481</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="but_zoom_in">
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>105</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(92, 92, 92);</string>
      </property>
      <property name="text">
       <string>Zoom_in</string>
      </property>
      <property name="iconSize">
       <size>
        <width>33</width>
        <height>33</height>
       </size>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="but_zoom_out">
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>105</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(92, 92, 92);</string>
      </property>
      <property name="text">
       <string>Zoom_out</string>
      </property>
      <property name="iconSize">
       <size>
        <width>33</width>
        <height>33</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Zoom:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_zoom">
        <property name="text">
         <string>1x</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="but_build_graf">
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>105</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(92, 92, 92);</string>
      </property>
      <property name="text">
       <string>Build</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="line_text_graf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="checkBox_accursy">
      <property name="text">
       <string>Более точное построение</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_point">
      <property name="text">
       <string>Построение графика точками</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
